docs/samples/browser-plugin-meetings/app.js

```diff
@@ -427,19 +427,27 @@ function refreshCaptcha() {
 meetingsListElm.onclick = (e) => {
   selectedMeetingId = e.target.value;
   const meeting = webex.meetings.getAllMeetings()[selectedMeetingId];
+  const selectedMeetingType = createMeetingSelectElm.options[createMeetingSelectElm.selectedIndex].innerText;
+  console.log("MEETING TYPE AAAA: ",selectedMeetingType);
 
   if (meeting && meeting.passwordStatus === 'REQUIRED') {
     meetingsJoinPinElm.disabled = false;
     verifyPasswordElm.disabled = false;
     document.getElementById('btn-join').disabled = true;
     document.getElementById('btn-join-media').disabled = true;
   }
-  else if (meeting && meeting.passwordStatus === 'UNKNOWN') {
+  else if (meeting && (meeting.passwordStatus === 'UNKNOWN' && (selectedMeetingType !== "One to One Meeting" && selectedMeetingType!=="Email" && selectedMeetingType!=="PERSON ID" ))) {
```

Comment:
The only value accepted here is SIP. Can we remove the multiple `not equals` and add just SIP
```suggestion
  else if (meeting && (meeting.passwordStatus === 'UNKNOWN' && selectedMeetingType == "SIP") {
```

Reply:
Hii @Shreyas281299 the selectedMeetingType is regarding the option that we selected in html tags
 <option value="">Person ID</option>
                      <option value="">SIP URI</option>
                      <option value="CONVERSATION_URL">Conversation URL</option>
                      <option value="">One to One Meeting</option>
based on the option above we selected it will take innerHtml value like SIP URI or CONVERSATION URL or One to One Meeting so as it is for locus call we can do with Email or Person Id or One to One Meeting so i had keep the condition.

Reply:
Hii @Shreyas281299 the selectedMeetingType is regarding the option that we selected in html tags
 <option value="">Person ID</option>
                      <option value="">SIP URI</option>
                      <option value="CONVERSATION_URL">Conversation URL</option>
                      <option value="">One to One Meeting</option>

Reply:
`Hii @Shreyas281299 the selectedMeetingType is regarding the option that we selected in html tags
 <option value="">Person ID</option>
                      <option value="">SIP URI</option>
                      <option value="CONVERSATION_URL">Conversation URL</option>
                      <option value="">One to One Meeting</option>
based on the option above we selected it will take innerHtml value like SIP URI or CONVERSATION URL or One to One Meeting so as it is for locus call we can do with Email or Person Id or One to One Meeting so i had keep the condition.`

Reply:
@Shreyas281299 Updated code